plugins/statistics.koplugin/main.lua

```diff
@@ -1048,6 +1059,22 @@ function ReaderStatistics:getPageTimeTotalStats(id_book)
     return total_pages, total_time
 end
 
+function ReaderStatistics:usePageMapForStats()
+    if not self.ui.pagemap then
+        return false
+    elseif not self.ui.pagemap.has_pagemap then
+        return false
+    end
+
+    if self.ui.doc_settings:has("pagemap_use_page_labels")  then
+        return self.ui.doc_settings:isTrue("pagemap_use_page_labels")
+    elseif G_reader_settings:isTrue("pagemap_use_page_labels") then
+        return true
+    end
```

Comment:
Can this be used instead?
https://github.com/koreader/koreader/blob/475280abe409a8ac1c074dbd4d694c89b18c4a5b/frontend/apps/reader/modules/readerpagemap.lua#L271-L273